packages/atomic/project.json

```diff
@@ -1 +1,264 @@
-{}
+{
```

Comment:
This whole file might be a pain to maintain or reverse engineer if something unexpected happens. 

Why not incrementally add these settings with the low hanging fruits like the files we copy (80/20 rule)?

Reply:
Most of the complexity comes from targets like "build:stencil" and "build:storybook", so you're right that going for low-hanging fruits like copying would make it simpler. However, these two steps take a long time to run, and therefore save a much more significant amount of time when cached.

I can incrementally add these changes, just like I didn't put as much effort in `dev` and `watch` scripts yet, but if we end up eventually adding the bigger/longer steps, we'll still end up with a config that's just as complex.

Using proper executors should significantly simplify the config since we wouldn't need to define their inputs/outputs (AFAIK).

Reply:
Update: I just found out that Nx supports "plugins", which allow us to dynamically generate `project.json` file contents (see [Project Inference](https://nx.dev/recipes/advanced-plugins/project-inference-plugins)). This could help us reduce repetition and write our configuration as code.